tracks/csharp/exercises/space-age/mentoring.md

```diff
@@ -0,0 +1,86 @@
+### Reasonable solutions
+
+#### using `Dictionary`
+
+```csharp
+using System.Collections.Generic;
+
+public class SpaceAge
+{
+    private readonly double earthYear;
+    private const double OrbitalPeriodEarth = 31_557_600.0;
+
+    private static Dictionary<string, double> conversionFactor 
+            = new Dictionary<string, double>()
+    {
+        {"Mercury", 0.2408467},
+        {"Venus",   0.61519726},
+        {"Mars",    1.8808158},
+        {"Jupiter", 11.862615},
+        {"Saturn",  29.447498},
+        {"Uranus",  84.016846},
+        {"Neptune", 164.79132},
+        {"Earth",   1.0}
```

Comment:
Minor nit: maybe move earth between venus and mars? To reflect its position in the solar system?

Reply:
sure, I will do this